### PR TITLE
Update aspnet50/aspnetcore50 => dnx50/dnxcore50.

### PR DESCRIPTION
**Pull request by NTaylorMullen**
Prepping to be merged once ready.

### CODE REVIEW

**Comment by msotclas** [comment]
Hi __@NTaylorMullen__, I'm your friendly neighborhood Microsoft Open Technologies, Inc. Pull Request Bot (You can call me MSOTBOT). Thanks for your contribution!
    <p>
        It looks like you're a Microsoft contributor (Taylor Mullen). If you're full-time, we DON'T require a Contribution License Agreement. If you are a vendor, please DO sign the electronic Contribution License Agreement. It will take 2 minutes and there's no faxing! https://cla2.msopentech.com.
    </p>

TTYL, MSOTBOT;

**Comment by muratg** [comment]
:shipit:

File: src/kvm.cmd [hunk 1]
```diff
@@ -1,8 +1,8 @@
-@Echo off
+﻿@Echo off
 
 set KVM_CMD_PATH_FILE="%USERPROFILE%\.k\temp-set-envvars.cmd"
 
-PowerShell -NoProfile -NoLogo -ExecutionPolicy unrestricted -Command "[System.Threading.Thread]::CurrentThread.CurrentCulture = ''; [System.Threading.Thread]::CurrentThread.CurrentUICulture = '';$CmdPathFile='%KVM_CMD_PATH_FILE%';& '%~dp0kvm.ps1' %*"
+PowerShell -NoProfile -NoLogo -ExecutionPolicy unrestricted -Command "[System.Threading.Thread]::CurrentThread.CurrentCulture = ''; [System.Threading.Thread]::CurrentThread.CurrentUICulture = '';$CmdPathFile='%KVM_CMD_PATH_FILE%';& '%~dp0dnvm.ps1' %*"
 
 IF EXIST "%KVM_CMD_PATH_FILE%" (
```
Comment (muratg) [inline src/kvm.cmd]: Needs to change?

File: src/kvm.sh [hunk 2]
```diff
@@ -1,26 +1,26 @@
-# kvm.sh
+﻿# dnvm.sh
 # Source this file from your .bash-profile or script to use
 
 # "Constants"
 _KVM_BUILDNUMBER="{{BUILD_VERSION}}"
```
Comment (muratg) [inline src/kvm.sh]: KVM?

**Pull request closed without merging**